meetings/2021-07/july-14.md

```diff
@@ -0,0 +1,1028 @@
+
+SFC: I like the term "concept" but it almost sounds more like something at Stage 1. We currently use the word "proposal" for Stage 1, which is misleading, because a proposal is the thing that goes through all the stages. So I feel like that word is kind of overused. So I like the word "concept" but that sounds kind of more descriptive of Stage 1. Stage 0 is more like an "idea", maybe, like an "idea" at Stage 0 and a "concept" at Stage 1. Just thought I'd throw that out there.
+
+HHM: Concept is basically an abstract idea, right?
+
+SFC: So, maybe concept is 0 and idea is 1. If we're changing the names, I just feel like stage 1 being called "proposal" is kind of misleading. So, we may as well fix both? That's all; you can move on to the next discussion topic.
+
+CM: I think while the proposal is well-intentioned, I think it kind of misses the point of the original terminology, the idea of a straw man or straw person was something which is kind of initially flimsy, which is what things often are at the beginning when they haven't been fleshed out. And the whole idea is something which is in a flimsy or weak state. And then as you work on it, you make it stronger and more sturdy and more sound. A “concept” is just an idea. The whole idea behind "straw person" or whatever word you use for it, is the idea that you're capturing there, is that it is quite possibly very flawed: it's very flimsy, it's not necessarily all the way thought through and should be regarded such, meaning it'll be easy to attack and therefore the thing to do is not just blindly attack it, because anybody can do that, but to have constructive contributions that can shore it up and make it stronger, make it more sound. And that sort of evolution from something, which starts as a sort of weak proto-notion to something which is more fleshed out is, I think, lost if you switch to some abstraction like "concept".
+
+AKI: Thanks for that chip. That's a good point.
+
+SYG: So maybe I misunde the actual intention of this proposal. Is the proposal to change the process document or to kind of get us on the same page to try to stop using the word straw person?
+
+AKI: I think it's both.
+
+SYG: I see. It seems like it would be clear for the process document to perhaps just drop any mnemonic.and just, we just refer to Stage 0 through 4. As for getting us on the same page, that seems like a taller order but because I have nothing against "concept". Sorry, that was not very clear. I'm proposing that instead of spending time in plenary to debate whether "concept" is a good replacement for strawperson, we could just drop it from the process document.
```

Comment:
```suggestion
SYG: I see. It seems like it would be clear for the process document to perhaps just drop any mnemonic just refer to Stage 0 through 4. As for getting us on the same page, that seems like a taller order but because I have nothing against "concept". Sorry, that was not very clear. I'm proposing that instead of spending time in plenary to debate whether "concept" is a good replacement for strawperson, we could just drop it from the process document.
```